development.md

```
@@ -38,75 +40,102 @@ docker compose logs backend
 ```bash
-docker compose watch
+cd backend
+uv run celery -A app.celery.celery_app flower --port=5555
 ```
 
-When deploying, for example in production, the main Traefik is configured outside of the Docker Compose files. For local development, there's an included Traefik in `docker-compose.override.yml`, just to let you test that the domains work as expected, for example with `api.localhost.tiangolo.com` and `dashboard.localhost.tiangolo.com`.
+Flower will be available at: http://localhost:5555
 
-## Docker Compose files and env vars
+> **Note:** If you start Flower before running any Celery workers, you may see warnings like:
+> ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)